integrated_channels/canvas/models.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+"""
+Database models for Enterprise Integrated Channel Canvas.
+"""
+
+from logging import getLogger
+
+from config_models.models import ConfigurationModel
+from simple_history.models import HistoricalRecords
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+from integrated_channels.integrated_channel.models import EnterpriseCustomerPluginConfiguration
+
+LOGGER = getLogger(__name__)
+
+
+# pylint: disable=feature-toggle-needs-doc
+@python_2_unicode_compatible
+class CanvasGlobalConfiguration(ConfigurationModel):
```

Comment:
These models are also not final in this PR, much of that work to figure out fields and complete these models will happen in upcoming subtask (s)